### PR TITLE
GRW-2370 - chore(PetBreedField/MixedBreed): added translations

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

Added translations for `PetBreedField` and `MixedBreed`

## Jira issue(s): [GRW-2370](https://hedvig.atlassian.net/browse/GRW-2370)

[GRW-2370]: https://hedvig.atlassian.net/browse/GRW-2370?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1923** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1923" target="_blank">Graphite</a> 
    * **PR #2002** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2002" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2002?utm_source=stack-comment).

File: apps/store/public/locales/en/cart.json [hunk 1]
```diff
@@ -1,6 +1,4 @@
 {
-  "ACCIDENT_OFFER_CONTENT": "Extend your protection with an accident insurance. Covers accidents, dental injuries, scars and much more. The insurance applies without a deductible.",
```
Comment (alebedev) [inline apps/store/public/locales/en/cart.json:2]: Judging by build error, should this key be removed?

Comment (guilhermespopolin) [reply apps/store/public/locales/en/cart.json:2]: Oh that's true. What's weird though is that removal is not directly related with this pr but part of `download-translation` script. I guess someone has deleted it in lokalise. Checking...

Comment (guilhermespopolin) [reply apps/store/public/locales/en/cart.json:2]: Tried to get back that `ACCIDENT_OFFER_CONTENT` but lokalise keeps telling me it already exists, despite the fact that I can't find it in the system. Got it back with a new name 🤷🏻

**Review by alebedev — APPROVED** [review]
Looks good

**Pull request merged**